### DISCUSSION TITLE
GE2 | Μέρος Α – Ανάλυση (Ομαδική με όλη την τάξη)

### DISCUSSION THREAD

**Discussion by IoannisTheodoropoulos** [sdy60/syllabus#110] [Show and tell]
Καλησπέρα σας,

αν θέλετε μπορούμε να χρησιμοποιήσουμε αυτη την ενότητα (https://github.com/SDY60-2023/webring/discussions/43) για την ενδοεπικοινωνία μας όσον αφορά το Μέρος Α - Ανάλυση (Ομαδική με όλη την τάξη)!

**Comment by std133524** [comment]
Υπάρχει μια σχετική συζήτηση  πριν ένα χρόνο περίπου για το ίδιο θέμα : https://github.com/orgs/sdy60-2022/discussions/40. Δεν ξέρω αν βοηθά αλλά καλύπτει πάνω κάτω το ίδιο θέμα
